01-Fundamentals-part-1/challenge_1_Conditionals.js

```diff
@@ -0,0 +1,53 @@
+//Problem: Create 3 functions that would calculate a discount based on gender ("discountCalc1"
+//- using if/else conditions, "discountCalc2" - using switch statement, "discountCalc3" - using ternary operators).
+//The function would take two parameters -  price (number) and gender ("male", "female", "unknown") and return discounted price.
+//If gender is male, function applies 10% discount.
+//If gender is female, function applies 20% discount.
+//If gender is unknown, function applies 30% discount.
+
+const price = 100;
```

Comment:
Let's better put smth like `const price = prompt("Enter your price");`  so that I could test 3 functions return the same thing